borg/platform_windows.pyx

```diff
@@ -0,0 +1,295 @@
+from __future__ import absolute_import
```

Comment:
Found it. We need to add `language_level=3` to the cython_compiler() call in setup.py to enable PY3 source level by default.